web-ui/src/components/search-results/SearchBirthdayAnniversaryResults.jsx

```diff
@@ -15,26 +15,25 @@ import {
 
 const SearchBirthdayAnniversaryResults = ({
   hasSearched,
-  searchBirthdayResults,
-  searchAnniversaryResults,
+  results,
   anniversary,
```

Comment:
`anniversary` and `birthday` are boolean arguments that indicate which kind of data is described by the results array. I could replace these with a single boolean parameter, but then callers would have to explicitly pass `true` or `false` instead of just using those as attributes with no value.